wrapper/hosts.py

```diff
@@ -525,6 +539,25 @@ def validate_data(self, data):
                 hard_error('Missing mac address in one of network mappings')
         return data
 
+    def _check_ip_in_network(self, ipaddr, network):
```

Comment:
it would be nice to have at least a few tests for this and the following two functions